src/article_maker.cc

```diff
@@ -676,11 +663,11 @@ void ArticleRequest::bodyFinished()
                      </div>)" ),
           dictId,
           collapse ? R"(style="cursor:pointer;")" : "",
```

Comment:
Maybe we should change this `pointer` to something like `row-resize` or `ns-resize` to better repent the "Collapse & Expand".

https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Reply:
pointer means cickable.
![image](https://github.com/user-attachments/assets/4fa38878-3bfc-4128-9d0e-b6cc94146fd1)
*-resize means it can be resized which is not. The collapse area can not be resized.

I'll choose `pointer`